anni-common/src/models.rs

```diff
@@ -26,6 +28,10 @@ impl<'a> RawTrackIdentifier<'a> {
             },
         }
     }
+
+    pub fn copied(&'a self) -> Self {
```

Comment:
If it compiles after removing the `Clone` implementation, just replace it :)

Reply:
Actually it is hard because clone accepts arbitrary lifetime of self but we want to return a `'a`